ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -0,0 +1,168 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @error={{@model.errors.base}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+  <form.input
+    @value={{@model.type}}
+    @label={{t 'form.type.label'}}
+    readonly={{true}}
+    @disabled={{true}}
+  />
+
+  <form.input
+    @name='name'
+    @type='text'
+    @value={{@model.name}}
+    @label={{t 'form.name.label'}}
+    @helperText={{t 'form.name.help'}}
+    @error={{@model.errors.name}}
+    readonly={{false}}
+    as |field|
+  >
+    {{#if @model.errors.name}}
+      <field.errors as |errors|>
+        {{#each @model.errors.name as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.input>
+
+  <form.textarea
+    @name='description'
+    @type='text'
+    @value={{@model.description}}
+    @label={{t 'form.description.label'}}
+    @helperText={{t 'form.description.help'}}
+    @error={{@model.errors.description}}
+    disabled={{@model.isSaving}}
+    readonly={{false}}
+    as |field|
+  >
+    {{#if @model.errors.description}}
+      <field.errors as |errors|>
+        {{#each @model.errors.description as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.textarea>
+
+  <form.input
+    @name='issuer'
+    @type='text'
+    @value={{@model.attributes.issuer}}
+    @label={{t 'form.issuer.label'}}
+    @helperText={{t 'form.issuer.help'}}
+    @error={{@model.errors.attributes.issuer}}
+    readonly={{false}}
+    as |field|
+  >
+    {{#if @model.errors.attributes.issuer}}
+      <field.errors as |errors|>
+        {{#each @model.errors.attributes.issuer as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.input>
+
+  <form.checkbox
+    @label={{t 'form.disable_discovered_config_validation.label'}}
+    @helperText={{t 'form.disable_discovered_config_validation.help'}}
+    @checked={{@model.attributes.disable_discovered_config_validation}}
+    onChange={{fn
+      (mut @model.attributes.disable_discovered_config_validation)
```

Comment:
Today I learnt about [mut](https://api.emberjs.com/ember/3.26/classes/Ember.Templates.helpers/methods/mut?anchor=mut). Awesome!

Reply:
It's Ember magic that allows a value to be mutated somewhat declaratively, so that it we don't need to write a trivial action to do it.